readme.md

```diff
@@ -8,25 +8,78 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-[**rehype**][rehype] plugin to bridge or mutate to [**retext**][retext].
-
-Uses [`hast-util-to-nlcst`][to-nlcst] under the hood.
-See its documentation to learn how to ignore nodes.
+**[rehype][]** plugin to support **[retext][]**.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`unified().use(rehypeRetext, destination)`](#unifieduserehyperetext-destination)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Related](#related)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package is a [unified][] ([rehype][]) plugin to support [retext][].
+
+**unified** is a project that transforms content with abstract syntax trees
+(ASTs).
+**rehype** adds support for HTML to unified.
+**retext** adds support for natural language to unified.
+**hast** is the HTML AST that rehype uses.
+**nlcst** is the natural language AST that retext uses.
+This is a rehype plugin that transforms hast into nlcst to support retext.
+
+## When should I use this?
+
+This project is useful if you want to check natural language in HTML.
+The retext ecosystem has many useful plugins to check prose, such as
+[`retext-indefinite-article`][retext-indefinite-article] which checks that `a`
+and `an` are used correctly, or [`retext-readability`][retext-readability] which
+checks that sentences are not too complex.
+This plugins lets you use them on HTML documents.
+
+This plugin is unfortunately not able to apply changes by retext plugins (such
```

Comment:
```suggestion
This plugin is not currently able to apply changes by retext plugins (such
```

Reply:
Feels worse? At least `not currently` read more natural to me. But then I’m still 🤷‍♂️ on what it improves. (I do find it unfortunate.)

Reply:
I think it’s best to leave sentiment (such as `unfortunately`) out of technical documentation.

`currently` implies that it might be possible later. I think that should be avoided as well, unless there’s an open issue for it.

```suggestion
This plugin is not able to apply changes by retext plugins (such
```